deploy-board/deploy_board/webapp/host_views.py

```diff
@@ -19,6 +19,9 @@
 from helpers import environs_helper, agents_helper, autoscaling_groups_helper
 from helpers import environ_hosts_helper, hosts_helper
 from deploy_board.settings import IS_PINTEREST, TELETRAAN_HOST_INFORMATION_URL
+import brood
```

Comment:
Didn't notice this at first place. This is a Pinterest specific library. I would rather write a simple http client to query cmdb instead of adding a new library dependency.